app/models/program.rb

```diff
@@ -49,7 +51,7 @@ def confirmed
 #   validates :conference_id, presence: true, uniqueness: true
   validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 10 }
 
-  before_create :create_event_types
+  after_create :create_event_types
```

Comment:
I think the creation of an event type is an action that happens, so it should be logged, even though it is supposed to happen automatically for every conference.